src/api/app/controllers/source_controller.rb

```diff
@@ -785,8 +785,8 @@ def project_command_createpatchinfo
                                                    comment: params[:comment], force: params[:force])
   end
 
-  # POST /source/<project>/<package>?cmd=updatepatchinfo
-  def package_command_updatepatchinfo
+  # POST /source/<project>/<package>?cmd=update
```

Comment:
Keep in mind that this is our API. We should be more conservative with changing it. Even if it's ugly

Reply:
yes, definitely it shouldn't be changed. Strange that didn't break any test case. It would probably break the ```osc``` I will rollback this commit.

Reply:
That's because you adjusted the tests;-)